packages/sdk/package.json

```diff
@@ -73,23 +73,26 @@
     "@waku/proto": "^0.0.8",
     "@waku/utils": "0.0.20",
     "@waku/message-hash": "0.1.16",
-    "libp2p": "^1.8.1"
+    "async-mutex": "^0.5.0",
+    "libp2p": "^1.8.1",
+    "p-event": "^6.0.1"
   },
   "devDependencies": {
-    "@types/mocha": "^10.0.6",
     "@types/chai": "^4.3.11",
     "@rollup/plugin-commonjs": "^25.0.7",
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.2.3",
     "@rollup/plugin-replace": "^5.0.5",
+    "@types/mocha": "^10.0.9",
     "@waku/build-utils": "*",
-    "mocha": "^10.3.0",
-    "sinon": "^18.0.0",
-    "chai": "^4.3.10",
+    "chai": "^5.1.1",
     "cspell": "^8.6.1",
     "interface-datastore": "^8.2.10",
+    "mocha": "^10.7.3",
     "npm-run-all": "^4.1.5",
-    "rollup": "^4.12.0"
+    "rollup": "^4.12.0",
+    "sinon": "^19.0.2",
+    "ts-node": "^10.9.2"
```

Comment:
why is it added?

Reply:
indeed, don't know how this leaked in 👀 removed, thanks!